alf/networks/relu_mlp.py

```diff
@@ -112,9 +112,10 @@ def forward(self,
             requires_jac (bool): whether outputs input-output Jacobian.
             requires_jac_diag (bool): whetheer outputs diagonals of Jacobian.
         """
-        inputs = inputs.squeeze()
-        assert inputs.shape[-1] == self._input_size, \
-            ("inputs should has shape {}!".format(self._input_size))
+        if inputs.ndim == 1:
```

Comment:
The shape of inputs and output are not consistent when ndim==1. Should squeeze the output if ndim==1

Reply:
updated.